src/helpers/utils.ts

```diff
@@ -16,13 +16,18 @@ export const pick = <T extends object, K extends keyof T>(base: T, keys: K[]) =>
   return pickedObject
 }
 
-export const parseConfigFile = async <T>(baseConfig: T, configPath?: string) => {
+export const getConfig = async (configPath: string) => {
+  const configFile = await promisify(fs.readFile)(configPath, 'utf-8')
```

Comment:
NIT: Isn't it equivalent to this?
```suggestion
export const getConfig = (configPath: string) => {
  const configFile = fs.readFileSync(configPath, 'utf-8')
```

Agreed that it's not parallelizable anymore, but it was always used with `await` aka blocking anyways.
I may be missing something here.

I just feel like that it's used as blocking, but is promisified just for the kicks of it.
Could be simplified if synchronous IMO, WDYT @m-rousse ?

If we're looking for optimisation, then the file content should be cached.

Reply:
`readFileSync` blocks the thread while it waits for the result of the system call, whereas during an `await` the event loop is not blocked and asynchronous calls can be performed in other parts of the code (callbacks, events from other parts of the code and so on).
It is generally a good practice to use `await` rather than `*Sync` for this reason, though I agree it does not change much here.
Also it is not needed to use `promisify` as standard lib already has promisified versions of each functions:
```suggestion
import {promises as fs} from 'fs'

[...]

export const getConfig = async (configPath: string) => {
  const configFile = await fs.readFile(configPath, 'utf-8')
```

Reply:
Of course, it was a way to simplify the code by making it synchronous.
While being a good practice, it still depends on the context and I just wanted to trigger the discussion as some food for thought.

Not needed for this PR of course.